package.json

```diff
@@ -87,5 +87,9 @@
     "webpack-cli": "3.3.10",
     "webpack-node-externals": "1.6.0"
   },
+  "resolutions": {
```

Comment:
To keep using same version of @types/react and @type/react-dom
cf. https://qiita.com/azu/items/22e0726924f32cd667d0